docs/source/onboarding_resources/computing/index.rst

```diff
@@ -319,11 +339,66 @@ researchers do not disturb engineering workflow.
 
 .. _cluster_access_other:
 
-Other Useful Cluster Tips
--------------------------
+Aliases and Other Cluster Tips
+------------------------------
+
+**Aliases:**
+
+Often, it can be annoying to type the same information repeatedly everytime you 
+access the cluster. To help account for this you can create aliases. These are 
+short-hand commands for commonly typed things. 
+
+Here is a Hub page written by the `Cost Effectiveness team <https://hub.ihme.washington.edu/display/CE/Setting+up+cluster+access>`_ on how to set up aliases. 
+
+Here, we provide a few copy and paste aliases you can add to your bashrc file. Be 
+sure to update the names to match your project and username. Also, note that once 
+you include these you will need to restart your cluster connection for them to take 
+effect. 
+
+The aliases below are: 
+
+#. Starting a Juptyer notebook in your project's repository 
+#. Starting an srun session (note: you can change the memory or other parameters before saving)
+#. Checking on your current jobs on the cluster 
+
+.. code-block:: bash 
+  :linenos:
+
+  $ alias jupyter="sh /ihme/singularity-images/rstudio/shells/jpy_rstudio_sbatch_script.sh -e <INSERT_ENVIRONMENT_NAME> -c /ihme/code/central_comp/miniconda/bin/activate -t lab -d /ihme/code/<INSERT_USERNAME>/<INSERT_PROJECT_REPO> -A proj_simscience -p i.q" 
+  $ alias srun="srun --mem=5G -c 1 -A proj_simscience -p all.q --pty bash" 
```

Comment:
Do you and/or others use this alias, and does it work for you? It would make me a little nervous, because since your alias has the same name as the original command, there isn't an easy way to access the original command anymore (e.g. to run an srun with different arguments). But if folks tend to do this, ignore me!

Reply:
I did test this method before adding (although I was not using it before) and it works! I would be interested if others use this or if there is a different, more flexible approach. I could also build out the below line about how to view the alias in your terminal to include copying and pasting the command with edits.

Reply:
Ahh I see what you mean about the repeating commands. Let me make this is a bit more explicit in terms of naming the aliases to avoid the issue.